src/common/feedGenerator.ts

```diff
@@ -124,20 +124,22 @@ const getRawFiltersData = async (
       qb
         .select(['"advancedSettingsId"', 'enabled'])
         .from(FeedAdvancedSettings, 't')
-        .where('"feedId" = $1'),
+        .where('"feedId" = $1')
+        .andWhere('"userId" = $2'),
     ),
     rawFilterSelect(con, 'tags', (qb) =>
       qb
         .select(['"keywordId"', 'status'])
         .from(ContentPreference, 't')
         .where('"feedId" = $1')
-        .andWhere(`type = '${ContentPreferenceType.Keyword}'`)
-        .andWhere('"userId" = $2'),
+        .andWhere('"userId" = $2')
+        .andWhere(`type = '${ContentPreferenceType.Keyword}'`),
```

Comment:
Are these just changing positions?

Reply:
yes i wanted to align all queries to first filter by feedId and userId so it will be easier to scan if we missed it somewhere

Reply:
💯